inc/Engine/Optimization/RUCSS/Frontend/APIClient.php

```diff
@@ -92,6 +92,7 @@ public function get_queue_job_status( $job_id, $queue_name, $is_home = false ) {
 		$result = json_decode( $this->response_body, true );
 		if ( key_exists( 'code', $result ) && 401 === $result['code'] ) {
 			set_transient( 'wp_rocket_no_licence', true, WEEK_IN_SECONDS );
+			$this->options->set( 'remove_unused_css', 0 );
```

Comment:
This only updates the value in the class storage, but not in the DB. To do so, you need to instantiate the `WP_Rocket\Admin\Options` class, and do something like the following:
```
$this->options->set( 'remove_unused_css', 0 );
$this->options_api->set( 'settings', $this->options->get_options() );
```

An alternative is to use `update_rocket_option(  'remove_unused_css', 0 );`